release-lib-master.Jenkinsfile

```diff
@@ -150,6 +151,42 @@ pipeline {
         }
       }
     }
+    stage('clean up and prepare folder again'){
+      steps{
+        script{withCredentials([
```

Comment:
Could we pull these from an openshift secret instead? I think we should start doing that where possible, to make our Jenkins easier to stand up / redeploy.